entrypoint.d/50-config-generate

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "${CONFIG_BUILD}" = true ]; then
```

Comment:
This env will never be available at run time (notice it is an `ARG`, not an `ENV` in the Dockerfile).
Instead, you can just check for existence of the file indicated in `$OPENERP_SERVER`.